tests/e2e/smoke/common.go

```diff
@@ -133,13 +133,13 @@ func CreateOCR2Jobs(
 	nkb []NodeKeysBundle,
 	mockserver *client.MockserverClient,
 	ocr2 contracts.OCRv2,
-	validator contracts.OCRv2DeviationFlaggingValidator) error {
+	store contracts.OCRv2Store) error {
 	relayConfig := map[string]string{
 		"nodeEndpointHTTP":   "http://sol:8899",
 		"nodeEndpointWS":     "ws://sol:8900",
 		"stateID":            ocr2.Address(),
-		"transmissionsID":    ocr2.TransmissionsAddr(),
-		"validatorProgramID": validator.ProgramAddress(),
+		"transmissionsID":    store.TransmissionsAddress(),
+		"validatorProgramID": store.ProgramAddress(),
```

Comment:
@aalu1418 I didn't rename this spec param so there's less breakage, but maybe we should do that anyway?